ods_ci/tests/Resources/Files/AppsInfoDictionary.json

```diff
@@ -244,7 +244,7 @@
                    },
                    "8": {
                          "text": "tutorial",
-                         "url": "https://docs.pachyderm.com/latest/getting_started/beginner_tutorial/",
+                         "url": "https://docs.pachyderm.com/latest/getting-started/beginner-tutorial/",
                          "matching": ""
                    },
                    "9": {
```

Comment:
these changes are not required, only the json file with _latest suffix should be changed. This one is used for retrocompatibility with previous version of RHODS